_layouts/berlin_event.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+---
+<style>h1, .main-wrapper h2, h3 {text-align: left; font-weight: bold;}</style>
```

Comment:
Any particular reason to not center headers like the other pages?

Reply:
yeah, its no big deal to me, we have those centered styles particularly for the main page headers. 

I wonder if maybe this could be a different layout (instead of default) with scoped header styles.